src/Shared/Components/Icon.js

```diff
@@ -32,7 +32,7 @@ import Octicons from "react-native-vector-icons/Octicons";
 import Zocial from "react-native-vector-icons/Zocial";
 import SimpleLineIcons from "react-native-vector-icons/SimpleLineIcons";
 
-const Icon = ({ iconStyle, iconName, iconSize, iconColor, style }) => {
+const Icon = ({ iconStyle, iconName, iconSize, iconColor, style, onPress }) => {
```

Comment:
Can you pass the onPress to all the icon options below?

Reply:
Done.

Reply:
Can you also add the onPress prop to the definition above?

Reply:
Done